### DISCUSSION TITLE
How to configure terminal shell prompt?

### DISCUSSION THREAD

**Discussion by lypwig** [lapce/lapce#3608] [General]
I use zsh but my Lapce terminal is bash, how to configure it to use zsh instead?

**Comment by jm-observer** [comment]
![setting_2](https://github.com/user-attachments/assets/379d838f-6f7c-4779-8810-6277bb481022)
![setting_1](https://github.com/user-attachments/assets/7a4c12ec-8a19-437e-9433-e74b7e0df071)

**Reply by lypwig** [reply]
Thank you!
I didn't know about this settings file.
Where am I supposed to find all the available options in the settings file that are not accessible via the setting ui?

**Reply by jm-observer** [reply]
i don't know.i just know this setting

**Reply by panekj** [reply]
best way is to look at https://github.com/lapce/lapce/blob/master/defaults/settings.toml and other files in https://github.com/lapce/lapce/tree/master/defaults
unfortunately looking up default settings in lapce is not that easy

**Comment by lamrongol** [comment]
I note the procedure:
1. Type `:Open Settings File` on command center
2. Write following setting(if you want to set `pwsh.exe`(PowerShell 7))
```
[terminal.profiles.default]
command = "C:\\Program Files\\PowerShell\\7\\pwsh.exe"
```

And following issues may be solved by this:
https://github.com/lapce/lapce/issues/2601 https://github.com/lapce/lapce/issues/2265 https://github.com/lapce/lapce/issues/150 https://github.com/lapce/lapce/issues/2892

Edit:
It seems that Lapce is not suited for Windows. I have a plan to change OS to Linux next time, I will try Lapce again at the time.
